packages/ic-response-verification/src/error.rs

```diff
@@ -15,63 +17,10 @@ pub enum ResponseVerificationError {
     #[error(r#"Failed to parse url: "{0}""#)]
     MalformedUrl(String),
 
-    /// The hash tree was malformed and could not be parsed correctly
```

Comment:
Moved to `packages/ic-certificate-verification/src/error.rs` and `packages/ic-cbor/src/error.rs`